README.md

```diff
@@ -1,12 +1,11 @@
 # Ruby Tricks
 
-[![Join the chat at https://gitter.im/franzejr/ruby-tricks](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/franzejr/ruby-tricks?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
I checked the rake task, it seems to include the gitter badge. I'm guessing this was not true a few days ago. Anyway, you should be able to rebase against master, run the rake task (it will populate the gitter badge now) and then squash...